<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QDockWidget" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>111</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>P&amp;layer controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="PlayerButton" name="btnRew">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Seeks five seconds backward</string>
        </property>
        <property name="whatsThis">
         <string>Seeks five seconds backward</string>
        </property>
        <property name="text">
         <string>Rew</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PlayerButton" name="btnPausePlay">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Plays or pauses a current playing song</string>
        </property>
        <property name="whatsThis">
         <string>Plays or pauses a current playing song</string>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
        <property name="shortcut">
         <string>Alt+P</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PlayerButton" name="btnStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Stops the currently playing song</string>
        </property>
        <property name="whatsThis">
         <string>Stops the currently playing song</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PlayerButton" name="btnFwd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Seeks five seconds forward</string>
        </property>
        <property name="whatsThis">
         <string>Seeks five seconds forward</string>
        </property>
        <property name="text">
         <string>Fwd</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblTimeCur">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Liberation Mono</family>
          </font>
         </property>
         <property name="toolTip">
          <string>Playing time</string>
         </property>
         <property name="whatsThis">
          <string>Time in minutes:seconds.tenth of milliseconds of how long the song has been played.</string>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Liberation Sans'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;---&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="seekSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblTimeRemaining">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Liberation Mono</family>
          </font>
         </property>
         <property name="toolTip">
          <string>Remaining time</string>
         </property>
         <property name="whatsThis">
          <string>Remaining time in minutes:seconds.tenth of milliseconds</string>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Liberation Sans'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;---&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/images/dryicons_clock.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlayerButton</class>
   <extends>QPushButton</extends>
   <header>playerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
